radicle/src/profile.rs

```diff
@@ -13,11 +13,23 @@
 use std::path::PathBuf;
 use std::{env, io};
 
+use thiserror::Error;
+
 use crate::crypto::{KeyPair, PublicKey, SecretKey, Signature, Signer};
-use crate::keystore::{Error, UnsafeKeystore};
+use crate::keystore::UnsafeKeystore;
 use crate::node;
 use crate::storage::git::Storage;
 
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error(transparent)]
+    Io(#[from] io::Error),
+    #[error(transparent)]
+    KeyStore(#[from] crate::keystore::Error),
+    #[error("no profile found at the filepath \"{0}\"")]
```

Comment:
```suggestion
    #[error("no profile found at the file path {0:?}")]
```

Reply:
(The debug output using `:?` will add quotes to strings and paths)

Reply:
I believe using the string version is still better.  Its a direct way of showing whats happening with the cost of a few extra characters.  Its also consistent with how other types are handled when reporting errors.  Adding inconsistencies makes the code harder to understand.

Reply:
If you search the codebase for an escaped double quote, I don't think you'll find one :)
I generally avoid using escape sequences because they add noise and there's almost always a way to do without (single quotes, back-ticks or `:?`).

Reply:
Ya, I usually opt for `'{0}'`. They're also useful for needing to copy-paste values. TIL about `:?` working like that with paths :)

Reply:
Opted for single ticks.

"TIL about :? working like that with paths"

Indeed it'd help newcomers to the code base if we opt for single ticks over `:?`.  Its a small extra detail they'd need to know, but death by a thousand paper cuts and all.